assets/css/styles.css

```
@@ -161,6 +161,14 @@ input[type="search"]::-webkit-search-decoration {
   box-shadow: 0 4px 12px rgba(0,0,0,.06);
 }
 
+/* ─── 12a. Color palette card hover ──────────────────────────────────────────*/
```

Stylelint (17.12.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)